src/types/felt252_dict_entry.rs

```diff
@@ -36,12 +35,12 @@ pub fn build<'ctx>(
     _metadata: &mut MetadataStorage,
     _info: WithSelf<InfoAndTypeConcreteType>,
 ) -> Result<Type<'ctx>> {
+    // TODO: Custom drop (dup?) for dict entries.
```

Comment:
is this TODO still relevant? same for the other todo

Reply:
Maybe. The dict entry owns the dict, so it may need to be dropped, but I'm not sure it's possible to drop a dict entry (it's handled internally). Let me try and break it so that it drops an entry and see what happens.

Also, where is the other drop?

Reply:
Apparently you can neither drop nor dup a dict entry so those are not necessary.
What you can do, however, is return a dict entry from the entrypoint, but I haven't implemented it because it may never happen (it's internal stuff). It won't affect contracts in any way since they use serializaztion.